client/components/admin/AdminTeams/imports/AdminTeamModalProgress.jsx

```diff
@@ -54,6 +54,10 @@ class AdminTeamModalProgress extends Component {
     const { team } = this.props;
     const { puzzles } = team;
 
+    if ((puzzles == undefined) || !team.hasBegun) {
```

Comment:
I believe generally `===` should be preferred unless you want JS to do some funky typecasting for you. Probably fine as-is, but just a weird quirk to know about

Reply:
fixed